cms/envs/common.py

```diff
@@ -1811,9 +1811,21 @@
 
     # alternative swagger generator for CMS API
     'drf_spectacular',
+
     'openedx_events',
+
+    # Learning Core Apps, used by v2 content libraries (content_libraries app)
+    "openedx_learning.core.components.apps.ComponentsConfig",
+    "openedx_learning.core.contents.apps.ContentsConfig",
+    "openedx_learning.core.publishing.apps.PublishingConfig",
```

Comment:
```suggestion

    # Learning Core Apps, used by v2 content libraries (content_libraries app)
    "openedx_learning.core.components",
    "openedx_learning.core.contents",
    "openedx_learning.core.publishing",
```

(non-blocking nit)

Django lets you abbreviate these when your app contains an `apps.py` with a lone/default app config.